src/Context/Video/Infrastructure/Symfony/Bundle/DependencyInjection/CodelyTvVideoExtension.php

```diff
@@ -11,6 +14,10 @@ class CodelyTvVideoExtension extends Extension
 {
     public function load(array $config, ContainerBuilder $container)
     {
+        $container->registerForAutoconfiguration(CommandHandler::class)->addTag('codely.video.command');
+        $container->registerForAutoconfiguration(QueryHandler::class)->addTag('codely.video.query');
+        $container->registerForAutoconfiguration(DomainEventSubscriber::class)->addTag('codely.video.subscriber');
```

Comment:
We're mapping the abstract interfaces such as `CommandHandler` to specific tags like the `codely.video.command` one. Moreover, we're having to do this mapping on every single Bounded Context Bundle.

What do you think about creating a shared bundle in order to be agnostic of the different bounded contexts and add there the generic mappings? This way we could avoid having to define this mappings each time we create a bounded context, doesn't we?